valtimo-dependencies/build.gradle

```diff
@@ -53,18 +53,26 @@ dependencies {
         api("com.ritense.valtimo:form-link:${projectVersion}!!")
         api("com.ritense.valtimo:form:${projectVersion}!!")
         api("com.ritense.valtimo:keycloak-iam:${projectVersion}!!")
+        api("com.ritense.valtimo:local-document-generation:${projectVersion}!!")
+        api("com.ritense.valtimo:milestones:${projectVersion}")
         api("com.ritense.valtimo:notes:${projectVersion}!!")
         api("com.ritense.valtimo:plugin-valtimo:${projectVersion}!!")
         api("com.ritense.valtimo:plugin:${projectVersion}!!")
+        api("com.ritense.valtimo:portaaltaak:${projectVersion}!!")
```

Comment:
`portaaltaak` is GZAC because it uses the Notificaties API and the Objecten API

Reply:
moved